pom.xml

```diff
@@ -64,7 +64,7 @@
     <properties>
         <revision>2.25</revision>
         <changelist>-SNAPSHOT</changelist>
-        <jenkins.version>2.263.1</jenkins.version>
+        <jenkins.version>2.331</jenkins.version>
```

Comment:
we should refrain using non LTS this prevents users use last version of the plugin....

Reply:
So? If and when there is some important feature/fix that cannot wait for the next LTS line, https://gist.github.com/jglick/86a30894446ed38f918050c1180483e2 is easy enough.

Reply:
why not but it's not something so simple and probably not obvious for the majority of the plugins maintainers.
probably worth some documentation somewhere.
how do that work with CD? 
does the cd tooling can detect he needs to generate something between 23343_kljffe0345 and 53343_kljloe0345?

Reply:
> probably worth some documentation somewhere.

Agreed.

> how do that work with CD?

The same Gist shows an example. Again, yes, deserves formal documentation and perhaps some tooling to make it easier and more discoverable.